opm/simulators/flow/BlackoilModelNldd.hpp

```diff
@@ -411,6 +413,10 @@ class BlackoilModelNldd {
         // Local Newton loop.
         const int max_iter = model_.param().max_local_solve_iterations_;
         const auto& grid = modelSimulator.vanguard().grid();
+        double damping_factor = 1.0;
+        std::vector<std::vector<double>> convergence_history;
```

Comment:
Maybe this could be `vector<vector<Scalar>>` to match the declaration of `resnorms` above?